js/bybit.js

```diff
@@ -3471,9 +3368,7 @@ module.exports = class bybit extends Exchange {
             const result = this.safeValue (response, 'result', {});
             return this.parseOrder (result, market);
         } else {
-            if (market === undefined) {
-                throw new ArgumentsRequired (this.id + ' fetchOrder() requires a symbol argument for ' + type + ' markets');
-            }
+            this.checkRequiredSymbol ('fetchOrder', symbol);
```

Comment:
`fetchOrder` is still using `privateGetSpotV3PrivateOrder` - so i think that part is not complete yet.

Reply:
Thanks @xmatthias. They support some v5 endpoints for normal account (https://bybit-exchange.github.io/docs/pilot-feature#normal-account-is-supported-by-v5-api).